### ISSUE TITLE
Ethereal Blocks Sponsor Kudos

### ISSUE THREAD

**Issue by kuhnchris** [gitcoinco/kudosbadges#263]
# Timeline
( @owocki ) ASAP for the hackathon

# Title
* Ethereal Summit (Ethereal Blocks)
* Gitcoin (Ethereal Blocks)
* Ethereum Foundation (Ethereal Blocks)
* Airswap (Ethereal Blocks)
* Bancor (Ethereal Blocks)
* Portis (Ethereal Blocks)
* eToro (Ethereal Blocks)
* GoodDollar (Ethereal Blocks)
* eToroX (Ethereal Blocks)
* Fortmatic (Ethereal Blocks)
* Status.IM (Ethereal Blocks)

# Design Inspiration
Use some ethereal branded sponsor logo kudos.
Logos:
* https://i2.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/ethereal-summit-logo-horizotal-02.png?resize=1024%2C256&ssl=1
* https://i0.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/gitcoin-logo-copy.png?resize=1024%2C370&ssl=1
* https://i1.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/Ethereum-Foundation-logo.png?resize=1024%2C335&ssl=1
* https://hackathons.gitcoin.co/wp-content/uploads/2019/08/LogoTypeAndMark__1_.svg
* https://i2.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/ethereal-summit-tel-aviv-2019-sponsor-bancor-1.png?w=369&ssl=1
* https://i2.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/07/portis_logo_large.png?resize=1024%2C312&ssl=1
* https://i2.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/ethereal-summit-tel-aviv-2019-sponsor-etoro.png?w=500&ssl=1
* https://i0.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/ethereal-summit-tel-aviv-2019-sponsor-good-dollar.png?w=500&ssl=1
* https://i1.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/Screen-Shot-2019-08-21-at-9.07.56-AM.png?resize=1024%2C406&ssl=1
* https://i2.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/Matic.png?w=458&ssl=1
* https://i1.wp.com/hackathons.gitcoin.co/wp-content/uploads/2019/08/status-1.png?w=710&ssl=1

# Metadata
> ```
> - name: <sponsor name> - Ethereal Blocks
>   description: Ethereal Blocks - Sponsor Kudos
>   artist: mladen
>   priceFinney: 1
>   numClonesAllowed: 100
>   to_address: 
>   can_be_sent_by: 
>   marketplace_listed: True
>   tags: 
>    - hackathon
>    - sponsor
>    - etherealblocks
> ```

# Down Payment
See @vs77bb 

# Gen0 Owner Address
See @vs77bb

**Comment by MladenPetronijevic** [comment]
![Untitled](https://user-images.githubusercontent.com/41165756/63943439-f4473a00-ca6f-11e9-83c0-ae1800d8b714.png)

**Comment by kuhnchris** [comment]
@vs77bb please check!

Looks great @MladenPetronijevic !

**Comment by owocki** [comment]
so dope!  

we also need the kudos.yaml info filled out to mint these.. who want to write that copy / submit that PR?

**Comment by kuhnchris** [comment]
You mean the duplicates áka
```
- name: GoodDollar - Ethereal Blocks
  description: Ethereal Blocks - Sponsor Kudos
  artist: mladen
  priceFinney: 1
  numClonesAllowed: 100
  to_address: 
  can_be_sent_by: 
  marketplace_listed: True
  tags: 
   - hackathon
   - sponsor
   - etherealblocks
```

etc.?

**Comment by owocki** [comment]
yeah, just having something i can paste into `kudos.yaml` would be great.

**Comment by owocki** [comment]
@MladenPetronijevic can you post the svgs pls?

**Comment by MladenPetronijevic** [comment]
[sponsor kudos.zip](https://github.com/gitcoinco/kudosbadges/files/3569331/sponsor.kudos.zip)
@owocki
